index.html

```diff
@@ -1048,9 +1048,13 @@ <h3>Identifiers</h3>
         <p>
 When expressing statements about a specific thing, such as a person, product,
 or organization, it is often useful to use some kind of identifier so that
-others can express statements about the same thing. Identifiers that others
-are expected to use when expressing statements about a specific thing MUST be
-expressed using the <code>id</code> <a>property</a>.
+others can express statements about the same thing. This specification defines
+the <code>id</code> <a>property</a> for identifiers. The <code>id</code>
+<a>property</a> is intended to unambiguously refer to an object such as a person,
+product, or organization. This allows for the expression of statements about
+specific things in the verifiable credential. If present, identifiers that
+others are expected to use when expressing statements about a specific thing
+MUST be expressed using the <code>id</code> <a>property</a>.
```

Comment:
I do not think this phrasing -- 

```
If present, identifiers that 
others are expected to use when expressing statements about a specific thing 
MUST be expressed using the <code>id</code> <a>property</a>.
```

-- expresses what was intended.  Flipping the word order around without changing what the sentence is saying --

```
If identifiers (that others are expected to use when expressing statements 
about a specific thing) are present, they MUST be expressed using the 
<code>id</code> <a>property</a>.
```

I think the intended statement is about `id` (not about identifiers intended for reuse) and ought to include something about cardinality.  Something more like: 

```
If the <code>id</code> <a>property</a> is present, it MUST express an 
identifier that others are expected to use when expressing statements 
about specific thing identified by that identifier; the <code>id</code> 
<a>property</a> MUST NOT have more than one value; and the value of the 
<code>id</code> <a>property</a> MUST be a URI.
```